scripts/mitochondria_m2_wdl/AlignAndCall.wdl

```diff
@@ -417,7 +415,7 @@ task M2 {
       gatk --java-options "-Xmx~{command_mem}m" Mutect2 \
         -R ~{ref_fasta} \
         -I ~{input_bam} \
-        ~{"--genotyping-mode GENOTYPE_GIVEN_ALLELES --alleles " + gga_vcf} \
+        ~{"--alleles " + gga_vcf} \
```

Comment:
The docker here is still 4.1.1.0, but the GGA arg removal was in 4.1.4.0 -- so the docker should be incremented, right?

Reply:
Huh, I'm confused because when I use the 4.1.4.0 docker I get an error: `A USER ERROR has occurred: genotyping-mode is not a recognized option`. But looking at the PR that deletes that code I totally agree it's 4.1.4.0 when that change was made. I suppose we're in the process of updating the version to something much more recent once Andrea's changes go in, so for now I'll update the gatk version to 4.1.4.0 and keep this change so everything will line up.